app/src/main/java/com/codesoom/assignment/application/AuthenticationService.java

```diff
@@ -0,0 +1,36 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.utills.JwtUtill;
+import io.jsonwebtoken.Claims;
+import org.springframework.stereotype.Service;
+
+/**
+ * 사용자 권한을 관리합니다.
+ */
+@Service
+public class AuthenticationService {
+    private final JwtUtill jwtUtill;
+
+    public AuthenticationService(JwtUtill jwtUtill) {
+        this.jwtUtill = jwtUtill;
+    }
+
+    /**
+     * 사용자가 로그인하면 아이디를 인코딩하여 리턴합니다.
+     *
+     * @return 인코딩된 아이디
+     */
+    public String login() {
+        return jwtUtill.encode(1L);
```

Comment:
이런 경우에는 놓치지 않도록 FIXME나 TODO 태그 주석을 달아주세요.
```suggestion
        // FIXME
        return jwtUtill.encode(1L);
```

Reply:
-  FIXME : 문제가 있는것이 확실하지만, 그걸 지금 당장 그것을 수정할 필요는 없을 때.
-  TODO : 당장 구현하기 어려운 일. 추후 해야할 일
태그 주석을 활용해야겠군요!

Reply:
네 `FIXME`나 `TODO` 외에도 `XXX` 같은 것도 있지만 잘 안 씁니다. 태그 주석을 쓰면 IDE에서도 따로 관리해주는 기능이 있을 거에요. 터미널에서 grep으로 검색하기도 쉽죠.